VisitMeet/visitmeet
# Product working, tests passing, frozen_string_literal added, and more

To fire it up, run these two commands.
Never keep db data in your developent app, so you can run `rake db:reset` at any time.
`bundle install`
`rake db:reset`

TO TEST IT:
Open new Terminal into your app's directory
`bin/rspec ` # all tests should pass

TO TAIL the tests:
Open new Terminal into your app's directory
`tail -f log/test.log` # tail your test log

and ..

TO BROWSE IT:
Open new Terminal into your app's directory
`bin/rails s` # fire up the server, visit your browser http://localhost:3000

TO TAIL the rails server development log:
Open new Terminal into your app's directory
`tail -f log/development.log` # tail your development log

and ..

TO CONSOLE IT:
Open new Terminal into your app's directory
`bin/rails c` # fire up a console in another Terminal

TO TAIL the rails console development log:
`tail -f log/development.log` # tail your development log
